src/blocks/app-and-main-menu/_fhi-main-menu.block.scss

```diff
@@ -34,35 +24,58 @@
             display: block;
             font-size: $fhi-font-size-3;
             margin-right: $fhi-container-padding-xs;
-            padding: $fhi-px * 5 $fhi-space-2 $fhi-space-1;
-
-            &:hover {
-                background-color: $fhi-bg-color-3;
-            }
+            padding: ($fhi-px * 5) 0 $fhi-space-1 $fhi-space-2;
         }
-        &__toggler-icon {
-            display: inline-block;
-            font-size: $fhi-px * 22;
-            line-height: 1;
+        &__toggler-close-icon {
+            display: none;
+        }
+        &__toggler-menu-icon {
+            background-size: ($fhi-px * 21) auto;
         }
         &__toggler-text {
             display: inline-block;
-            font-family: $fhi-font-family-sans-serif-medium;
             line-height: 1.3;
             margin-left: $fhi-space-2;
             vertical-align: top;
         }
         &__nav {
             margin-top: $fhi-space-6;
         }
+        &__nav-item {
+            width: 100%;
+        }
+        &__logo-gfx {
+            filter: brightness(1000%);
+        }
         &__collapse.show {
-            .nav-link:not(.active) {
+            .fhi-main-menu__nav-link {
+                background-color: transparent;
+                border-bottom: 0;
                 color: $fhi-white;
+                text-align: left;
+            }
+        }
+
+        // need the full class name and not just the BEM nested notation for specificity due to inheritance of the nav tabs styling
```

Comment:
I rather we do 
```
.fhi-main-menu {
    &__nav-link.nav-link {
```
That way the comment is unnecessary since `.nav-link` give info of why increased specifisity is needed. It breaks the rule about not styling on Bootstrap classes, but it's an edge case, and it gives more expressive code.